lib/spaceship/tunes/app_analytics.rb

```diff
@@ -0,0 +1,145 @@
+module Spaceship
+  module Tunes
+    class AppAnalytics < TunesBase
+      # @return (Integer) The app ID to grab analytics for
+      attr_accessor :apple_id
+
+      attr_mapping({
+        'adamId' => :apple_id
```

Comment:
Are there any other attributes in the responses from `client.time_series_analytics` that we would want to map?